### PR TITLE
Get the request id from the thread local variable and log it

### LINKED ISSUE #6: For micro services using the container logger log 3scale request_id
3scale creates a request id for every API call. When we make a Provision request we need to collect logs from all of our services (catalog, approval, topology) for that request. This requires logging the request id if it exists.

https://github.com/ManageIQ/manageiq-loggers/blob/606f727605b8577b9a1c5b3ee676dbfd2bc03947/lib/manageiq/loggers/container.rb#L36

### PR DESCRIPTION
**Pull request by mkanoor**
Fixes #6

### CODE REVIEW

**Comment by mkanoor** [comment]
@carbonin @Fryguy 
Please review

File: lib/manageiq/loggers/container.rb [hunk 1]
```diff
@@ -61,6 +62,10 @@ def thread_id
         def translate_error(level)
           SEVERITY_MAP[level] || "unknown"
         end
+
+        def request_id
+          Thread.current[:current_request]&.request_id || "unknown"
```
Comment (Fryguy) [inline lib/manageiq/loggers/container.rb]: I'm wondering if it's better to just let it be nil than having "unknown"

**Comment by Fryguy** [comment]
@carbonin Does this work for you?  Then on the ManageIQ side we have to have some way to represent a request for queue tasks I guess?  Part of the reason @mkanoor went down this path was so we could find a way to deal with #3 as well

**Comment by carbonin** [comment]
This is fine for now ... but why not do something like the bot (:trollface:) suggested in https://github.com/ManageIQ/manageiq-loggers/issues/6#issuecomment-486815024.

I'm sure you guys had a conversation about this, but I don't understand how going this route now makes it easier to solve #3 in the future.

**Comment by mkanoor** [comment]
@carbonin We need to get something in that will allow us to see the logs in Kibana based on the request id so we can track an order thru the system, we could update the logic later.

**Comment by carbonin** [comment]
Yeah, I figured there was a time element to this. So like I said, this is fine for now, but I'd rather hold off on integrating this with ManageIQ until we have something more flexible in place.

So @mkanoor for now if you make the update @Fryguy suggested then this looks good to me.

**Pull request merged**